packages/frontend/src/app/[id]/page.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import { Entity } from '@/common/interfaces'
+import { backendUrl, betterFetch } from '@/api'
+import { NavigationBreadcrumbs } from '@/common/components/navigation/Breadcrumbs'
+import Image from 'next/image'
+import { Text4xlBold, TextSm, TextSmBold } from '@/common/components/Typography'
+
+export default async function EntityDetailsPage(params: {
+  params: { id: string }
+}) {
+  const { id } = params.params
+  const entity: Entity = await getEntityById(id)
+
+  return (
+    <div className="flex flex-col w-full items-center">
+      <NavigationBreadcrumbs
+        breadcrumbs={[
+          { href: '/', title: 'Trusted Entities' },
+          ...(entity
+            ? [
+                {
+                  href: `/${id}`,
+                  title: entity.name ?? id,
+                },
+              ]
+            : []),
+        ]}
+      />
+      <div className="card rounded-2xl p-16 bg-white text-center max-w-[40rem] w-full">
+        <div className="flex flex-col gap-8">
+          {entity.id ? (
+            <>
+              <div className="flex items-center flex-col gap-4">
+                <Image
+                  src={entity.logo_url}
+                  width={101}
+                  height={101}
+                  alt="text"
+                />
+                <Text4xlBold>{entity.name}</Text4xlBold>
```

Comment:
We use 2xl everywhere (which is 24px) but on designs that title has 32px. Since there is no 32px font size in talewind I've used 36px (4xl) but I see  there is 30px (3xl) which is closer to designs. I'll change it to 3xl. When Marek will return back from vacation I will discuss it with him.

Reply:
Let's make it 2xl then, by introducing 3xl you're making it unnecessarily more complex.